.github/CONTRIBUTING.md

```diff
@@ -118,45 +119,83 @@ This project uses [Prettier](http://prettier.io/) for code formatting. Consider
 
 ## Commiting
 
-- Never use the `-m <msg>` / `--message=<msg>` flag to `git commit`;
+A commit message can consists of a **header**, **body** and **footer**. The header is the only mandatory part and consists of a type and a subject. The body is used to fully describe the change. The footer is the place to reference any issues or pull requests related to the commit. That said, we end with a template like this:
+
+```
+<type>: <subject>
+
+[optional body]
+
+[optional footer]
+```
+
+To ensure that a commit is valid, easy to read, and changelog-ready, we have a hook that lints the commit message before allowing a commit to pass. This linter verifies the following:
+
+- The header (first line) is the only mandatory part of the commit message.
+- The header should contains:
+  - A type:
+    - Must be lowercase;
+    - Must be one of:
+      - **chore**: A change that neither fix a bug or adds a feature;
```

Comment:
`adds a feature` in `chore` and `a new feature` in `feat` are confusing.

Reply:
Why?

`feat` = `feature`, adds a new feature.

`chore`, not a but fix nor a new feature.

Funny that it made me realize, I used the `feat` type in this PR/commit when it should have been `chore`.

Reply:
Personally I don't think it is confusing. 

The way it is worded, it is clear that `chore` is not supposed to be a feature.

Reply:
Personally I don't think it is confusing.

The way it is phrased, it is very clear that `chore` is not supposed to be a feature.

Reply:
Personally, i still think it's confusing, chore can fiz a bug or add a feature? So why exists the feature type, it can add a new feature too?

Reply:
Now i understood, sorry, i saw the `nor` now, i thought it was `or`. Bye.

Reply:
`A change that neither fix a bug or adds a feature;`

`neither` in this case indicates that whatever options come next will be false. So a `chore` is **not** a bug fix, **nor** a feature.